opensrp-pnc/src/main/java/org/smartregister/pnc/repository/PncVisitChildStatusRepository.java

```diff
@@ -17,9 +16,10 @@
 public class PncVisitChildStatusRepository extends BaseRepository implements PncGenericDao<Map<String, String>> {
 
     private static final String CREATE_TABLE_SQL = "CREATE TABLE " + PncDbConstants.Table.PNC_VISIT_CHILD_STATUS + "("
-            + PncDbConstants.Column.PncVisit.PARENT_BASE_ENTITY_ID + " VARCHAR NOT NULL, "
-            + PncDbConstants.Column.PncVisit.BASE_ENTITY_ID + " VARCHAR NULL, "
+            + PncDbConstants.Column.PncVisitInfo.PARENT_BASE_ENTITY_ID + " VARCHAR NOT NULL, "
```

Comment:
rename this to `pnc_visit_id` and update the constants

Reply:
done